scripts/scripts.js

```diff
@@ -65,7 +65,7 @@ const CONFIG = {
     il_en: { ietf: 'en-IL', tk: 'hah7vzn.css' },
     il_he: { ietf: 'he', tk: 'qxw8hzm.css', dir: 'rtl' },
     in_hi: { ietf: 'hi', tk: 'qxw8hzm.css' },
-    in: { ietf: 'en-GB', tk: 'hah7vzn.css' },
+    in: { ietf: 'en-IN', tk: 'hah7vzn.css' },
```

Comment:
Have you considered what other features use `ietf`? I think there was difficulty in the past about changing these values because of what they're used for. I can't quite remember what though, maybe georouting or analytics?

Reply:
Looks like caas cards broke here when they tried to update `la`: #130

Reply:
Good call, I'll go check all the places that it's used.

Reply:
CaaS cards appear to be working, this may have been resolved since last year.
https://bmarshal-in-lang--bacom--adobecom.hlx.live/in/resources/main 

Analytics updates are automatic, I couldn't identify any other issues.